pkl-cli/pkl-cli.gradle.kts

```diff
@@ -117,53 +248,108 @@ tasks.check {
 // To catch this and similar problems, test that Java executable starts successfully.
 val testStartJavaExecutable by tasks.registering(Exec::class) {
   dependsOn(javaExecutable)
-val outputFile = file("$buildDir/testStartJavaExecutable") // dummy output to satisfy up-to-date check
+  // dummy output to satisfy up-to-date check
+  val outputFile = layout.buildDirectory.dir("testStartJavaExecutable").get().asFile
   outputs.file(outputFile)
-  
+
   executable = javaExecutable.get().outputs.files.singleFile.toString()
   args("--version")
-  
+
   doFirst { outputFile.delete() }
-  
   doLast { outputFile.writeText("OK") }
 }
 
 tasks.check {
   dependsOn(testStartJavaExecutable)
 }
 
+val kernel32Init = listOf(
+  "org.msgpack.core.buffer.DirectBufferAccess",
+  "org.jline.nativ.Kernel32",
+  "org.jline.nativ.Kernel32${'$'}CHAR_INFO",
+  "org.jline.nativ.Kernel32${'$'}CONSOLE_SCREEN_BUFFER_INFO",
+  "org.jline.nativ.Kernel32${'$'}COORD",
+  "org.jline.nativ.Kernel32${'$'}FOCUS_EVENT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}INPUT_EVENT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}INPUT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}KEY_EVENT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}MENU_EVENT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}MOUSE_EVENT_RECORD",
+  "org.jline.nativ.Kernel32${'$'}SMALL_RECT",
+  "org.jline.nativ.Kernel32${'$'}WINDOW_BUFFER_SIZE_RECORD",
+).joinToString(",")
+
+fun createArchiveTasks(
+  exec: Provider<Exec>,
+  outputFile: File,
+  release: Boolean = enableRelease,
+) {
```

Comment:
The `pkl-cli` module now creates Gradle [distributions](https://docs.gradle.org/current/userguide/distribution_plugin.html) for the native CLI, in both zip and tar. These are the distributions I am posting to the PR for download.